docs/git.md

```diff
@@ -7,32 +7,70 @@ One of the most unique features of Simperby is its use of Git.
 You should have a basic understanding of
 
 1. Blockchain
-2. [Simperby Consensus and Governance](./protocol_overview.md)
+2. [Simperby Governance and Consensus](./protocol_overview.md)
 3. Git
 
 ## Summary
 
 1. **Every finalized data in a Simperby chain is stored in a Git repository.**
 2. It includes transactions, agendas, agenda proofs, chats, and blocks, linearly committed in the `finalized` branch.
-3. A Simperby node manages its own Git repository and provides a Git server of it to the node operator.
-4. The node operator can fetch the blockchain data, walk through the history, and check the diffs using the Git protocol.
-5. **Every to-be-finalized data is also managed in a Git repository.**
-6. All the pending agendas (waiting for approval) will be presented as multiple branches grown from the `finalized` branch.
-7. The node operator may create their own transaction as a commit and push to a particular branch which represents an agenda proposal.
-8. **Simperby functions as a general distributed Git repository**, that may contain any useful data for the organization. This is trivially achieved because **we take transactions as Git commits.** (This can be understood as exploiting the 'blockchain state' as a Git repository)
+3. **Every to-be-finalized data is also managed in a Git repository.**
+4. All the pending agendas (waiting for approval) and pending blocks (waiting for finalization) will be presented as multiple branches grown from the `finalized` branch.
+5. A Simperby node manages its own Git repository and provides a read-only Git server to the peers and the node operator.
+6. The node operator can fetch (literally `git fetch`)  the local server (that the Simperby node manages) to their own local clone, **walk through the history, and check the snapshots and diffs using whatever Git client they want**.
+7. Network participants (the **peers**) can also **fetch the blockchain data from each other and verify the commits, and synchronize** the blockchain state or pending proposals.
+8. The node operator may create their own transaction as a commit and push to a designated branch (`work`) to create **their own proposal to rebase on the `finalized` branch**.
```

Comment:
```suggestion
8. The node operator can create their own transactions as commits and push them to a designated branch (`work`) to create **a proposal to rebase on the `finalized` branch**.
```

Reply:
or I think 
-> may create each of their own transaction as a commit

Reply:
fixed

Reply:
fixed